README.md

```diff
@@ -4,7 +4,7 @@ Serves the Angular application, and then runs the TestCafe tests.
 
 ## Install
 ```bash
-$ npm install --save-dev @politie/angular-testcafe-builder
+$ npm install --save-dev angular-testcafe
```

Comment:
If we remove travis CI tests let's remove the badge in the title of README

Reply:
Should we do something with the warning?
![image](https://user-images.githubusercontent.com/12034551/115831100-d33c1580-a419-11eb-87f0-443e846bf212.png)

Reply:
We can. It's just a bad habit to use `any` everywhere, and we try to not overuse it with this rule in Framework